Imagine/Cache/Resolver/CacheResolver.php

```diff
@@ -230,10 +231,24 @@ protected function setDefaultOptions(OptionsResolverInterface $resolver)
             'index_key' => 'index',
         ));
 
-        $resolver->setAllowedTypes(array(
-            'global_prefix' => 'string',
-            'prefix' => 'string',
-            'index_key' => 'string',
-        ));
+        $allowedTypesList = array(
+          'global_prefix' => 'string',
+          'prefix' => 'string',
+          'index_key' => 'string',
+        );
+
+        if (version_compare(Kernel::VERSION_ID, '20600') >= 0) {
```

Comment:
Yes, but it's deprecated since 2.6 to be removed in 3.0 https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/OptionsResolver/OptionsResolver.php#L591